stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/options/MessageOptionsView.kt

```diff
@@ -38,32 +72,63 @@ public class MessageOptionsView : FrameLayout {
         val threadReplyIcon: Int,
         val copyText: String,
         val copyIcon: Int,
+        val editText: String,
+        val editIcon: Int,
+        val flagText: String,
+        val flagIcon: Int,
         val muteText: String,
         val muteIcon: Int,
         val blockText: String,
         val blockIcon: Int,
         val deleteText: String,
-        val deleteIcon: Int
+        val deleteIcon: Int,
+        val copyTextEnabled: Boolean,
+        val deleteConfirmationEnabled: Boolean,
+        val deleteConfirmationTitle: String,
+        val deleteConfirmationMessage: String,
+        val deleteConfirmationPositiveButton: String,
+        val deleteConfirmationNegativeButton: String
```

Comment:
Nit: there are a bunch of these long parameter / property lists on this PR where we could add a trailing commas to the last entry

Reply:
Added